content/applications/inventory_and_mrp/inventory/shipping_receiving/setup_configuration.rst

```diff
@@ -174,6 +177,55 @@ from the shipping carrier.
 .. seealso::
    :doc:`Invoice cost of shipping <setup_configuration/invoicing>`
 
+.. _inventory/shipping_receiving/shipping-route:
+
+Route on shipping method
+------------------------
+
+Optionally, set different warehouse delivery processes for a shipping method by configuring
+different :doc:`routes <daily_operations/use_routes>` for it.
+
+.. example::
+   Configuring multiple routes per shipping method is helpful for adjusting warehouse delivery
+   processes based on:
+
+   - speed (e.g., use :doc:`one-step delivery <daily_operations/receipts_delivery_one_step>` for
+     express shipping, or :doc:`two-step <daily_operations/receipts_delivery_two_steps>` for
+     standard shipping).
+   - international shipping (e.g., use :doc:`three-step delivery
+     <daily_operations/delivery_three_steps>` to prepare documents for customs)
+   - in-store pickup or home delivery: ship from the central warehouse, or pick from the store's
+     stock, depending on customer selection.
+
+To set up routes, go to :menuselection:`Inventory app --> Configuration --> Routes`. Click
+:guilabel:`New` or select the desired route.
+
+On the route form, in the :guilabel:`Applicable On` section, tick the :guilabel:`Shipping Methods`
+checkbox.
+
+.. figure:: setup_configuration/setup_configuration/shipping-route.png
+   :align: center
+   :alt: Routes form with the Shipping Methods checkbox selected.
+
+   Routes form with the Shipping Methods checkbox ticked.
+
+Then, go to :menuselection:`Inventory app --> Configuration --> Shipping Methods` and select the
+desired shipping method.
+
+On the shipping method form, in the :guilabel:`Routes` field, select the available fulfillment
+routes from the drop-down menu.
+
+.. note::
+   If the desired route is not selectable, check that the :guilabel:`Shipping Methods` option is
+   enabled in the route's :guilabel:`Applicable On` section.
```

Comment:
```suggestion
   If the desired route is not selectable, check that the *Shipping Methods* option is
   enabled in the route's *Applicable On* section.
```
opting for italics here because the user is currently on the Shipping Method form, and this field is being alluded to (not shown on the U/I at this *exact* moment).